src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -262,22 +268,64 @@ public void assertMatchingGenotypesFromGenomicsDB(File input, File expected, Loc
+        runGenotypeGVCFSAndAssertComparison(genomicsDBUri, expected, args, VariantContextTestUtils::assertVariantContextsHaveSameGenotypes, reference);
+    }
+
+    @Test
+    public void testMaxAltsToCombineInGenomicsDB() {
+        final File tempGenomicsDB = GenomicsDBTestUtils.createTempGenomicsDB(CEUTRIO_20_21_GATK3_4_G_VCF, new SimpleInterval("20", 1, 11_000_000));
+        final String genomicsDBUri = GenomicsDBTestUtils.makeGenomicsDBUri(tempGenomicsDB);
+        final List<String> args = new ArrayList<String>();
+        args.add("--"+GenotypeCalculationArgumentCollection.MAX_ALTERNATE_ALLELES_LONG_NAME);
+        args.add("3");
+        args.add("--" + GenomicsDBArgumentCollection.MAX_ALTS_LONG_NAME);
+        args.add("4");
+        runGenotypeGVCFSAndAssertCount(genomicsDBUri, args, 3, VariantContextTestUtils::assertVariantContextMaxAltAlleleCount, b37_reference_20_21);
+
+        args.clear();
+        args.add("--"+GenotypeCalculationArgumentCollection.MAX_ALTERNATE_ALLELES_LONG_NAME);
+        args.add("2");
+        args.add("--" + GenomicsDBArgumentCollection.MAX_ALTS_LONG_NAME);
+        args.add("20");
+        runGenotypeGVCFSAndAssertCount(genomicsDBUri, args, 2, VariantContextTestUtils::assertVariantContextMaxAltAlleleCount, b37_reference_20_21);
+    }
+
+    @Test(expectedExceptions = UserException.BadInput.class)
+    public void testGDBMaxAltsLessThanGGVCFsMaxAlts() {
+        final File input = CEUTRIO_20_21_GATK3_4_G_VCF;
+        final SimpleInterval interval =  new SimpleInterval("20", 1, 11_000_000);
+        final File tempGenomicsDB = GenomicsDBTestUtils.createTempGenomicsDB(input, interval);
+        final String genomicsDBUri = GenomicsDBTestUtils.makeGenomicsDBUri(tempGenomicsDB);
+        final List<String> args = new ArrayList<String>();
+        args.add("--"+GenotypeCalculationArgumentCollection.MAX_ALTERNATE_ALLELES_LONG_NAME);
+        args.add("20");
+        args.add("--"+GenomicsDBArgumentCollection.MAX_ALTS_LONG_NAME);
+        args.add("2"); // Too small max_alternate_alleles arg to GenomicsDB, should throw
+        File output = runGenotypeGVCFS(genomicsDBUri, null, args, b37_reference_20_21);
+    }
+
+    @Test(expectedExceptions = UserException.BadInput.class)
+    public void testGDBMaxAltsEqualsGGVCFsMaxAlts() {
+        final File input = CEUTRIO_20_21_GATK3_4_G_VCF;
+        final SimpleInterval interval =  new SimpleInterval("20", 1, 11_000_000);
+        final File tempGenomicsDB = GenomicsDBTestUtils.createTempGenomicsDB(input, interval);
+        final String genomicsDBUri = GenomicsDBTestUtils.makeGenomicsDBUri(tempGenomicsDB);
+        final List<String> args = new ArrayList<String>();
+        args.add("--"+GenomicsDBArgumentCollection.MAX_ALTS_LONG_NAME);
+        args.add("5");
+        args.add("--"+GenotypeCalculationArgumentCollection.MAX_ALTERNATE_ALLELES_LONG_NAME);
+        args.add("5"); // GenotypeGVCFs value needs to be at least one more, should throw
```

Comment:
"GenotypeGVCFs value needs to be at least one more" should be "GenomicsDB value needs to be at least one more"